generators/client/index.js

```diff
@@ -184,11 +184,6 @@ module.exports = class extends BaseGenerator {
                     this.baseName = baseName;
                 }
 
-                this.serviceDiscoveryType = configuration.get('serviceDiscoveryType') === 'no' ? false : configuration.get('serviceDiscoveryType');
```

Comment:
why is this removed wouldn't this fail truthy checks as the default value will be `no` and would pass the check?

Reply:
I guess this was code handling legacy .yo-rc files when we had 'no' as default option. Can you check with an old .yo-rc file before we changed from 'no' to false

Reply:
Tested with `no` and fixed the language generator in that case.  Everything else looks good